packages/@o3r/schematics/src/utility/loaders.ts

```diff
@@ -73,7 +73,14 @@ export function readPackageJson(tree: Tree, workspaceProject: WorkspaceProject)
  * @param projectType
  */
 export function getProjectFromTree(tree: Tree, projectName?: string | null, projectType?: 'application' | 'library'): WorkspaceProject & { name: string } | undefined {
-  const workspace = readAngularJson(tree);
+
+  let workspace: WorkspaceSchema;
+  try {
+    workspace = readAngularJson(tree);
+  } catch {
+    return undefined;
+  }
```

Comment:
use `getWorkspaceConfig` instead

```suggestion
  const workspace = getWorkspaceConfig(tree);
  if (!workspace) {
    return undefined;
  }
```

Reply:
indeed :)